devicons.py

```diff
@@ -147,6 +147,16 @@
     'Images'                           : '',
     'Publique'                         : '',
     'Vidéos'                           : '',
+# Portuguese
+    'Área de trabalho'                 : '',
+    'Documentos'                       : '',
+    'Downloads'                        : '',
+    'Dropbox'                          : '',
```

Comment:
This line is not adding much as the `Dropbox` key is already defined at line 136 and doesn't differ from the english definition.